admin/linux/build-appimage.sh

```diff
@@ -2,71 +2,82 @@
 
 set -xe
 
+export APPNAME=${APPNAME:-nextcloud}
+export BUILD_UPDATER=${BUILD_UPDATER:-OFF}
+export BUILDNR=${BUILDNR:-0000}
+
 mkdir /app
 mkdir /build
 
+export CC=/usr/bin/gcc
+export CXX=/usr/bin/g++
+
 #Set Qt-5.12
 export QT_BASE_DIR=/opt/qt5.12.10
 export QTDIR=$QT_BASE_DIR
 export PATH=$QT_BASE_DIR/bin:$PATH
 export LD_LIBRARY_PATH=$QT_BASE_DIR/lib/x86_64-linux-gnu:$QT_BASE_DIR/lib:$LD_LIBRARY_PATH
 export PKG_CONFIG_PATH=$QT_BASE_DIR/lib/pkgconfig:$PKG_CONFIG_PATH
 
-#Set APPID for .desktop file processing
-export LINUX_APPLICATION_ID=com.nextcloud.desktopclient.nextcloud
-
-#set defaults
-export SUFFIX=${DRONE_PULL_REQUEST:=master}
-if [ $SUFFIX != "master" ]; then
-    SUFFIX="PR-$SUFFIX"
+# Set defaults
+if [ "$BUILD_UPDATER" != "OFF" ]; then
+    BUILD_UPDATER=ON
 fi
 
-#QtKeyChain v0.10.0
+#QtKeyChain master
 cd /build
 git clone https://github.com/frankosterfeld/qtkeychain.git
 cd qtkeychain
 git checkout v0.10.0
 mkdir build
 cd build
 cmake -D CMAKE_INSTALL_PREFIX=/usr ../
-make -j4
-make install
+make -j$(nproc)
+make -j$(nproc) DESTDIR=/app install
+make -j$(nproc) install
+
 
 #Build client
 cd /build
 mkdir build-client
 cd build-client
 cmake -D CMAKE_INSTALL_PREFIX=/usr \
+    -D NO_SHIBBOLETH=1 \
     -D BUILD_TESTING=OFF \
-    -D BUILD_UPDATER=ON \
-    -DMIRALL_VERSION_SUFFIX=PR-$DRONE_PULL_REQUEST \
-    -DMIRALL_VERSION_BUILD=$DRONE_BUILD_NUMBER \
-    $DRONE_WORKSPACE
-make -j4
-make DESTDIR=/app install
+    -D BUILD_UPDATER=$BUILD_UPDATER \
+    -D QTKEYCHAIN_LIBRARY=/app/usr/lib/x86_64-linux-gnu/libqt5keychain.so \
```

Comment:
so we install either in `/app/usr` or we install in `/usr` but not both and then we do not know where to look for files